includes/glossary.md

```diff
@@ -35,7 +35,7 @@
 *[API]: Application Programming Interface. An agreed way for one system to communicate with another system.
 *[released outputs]: files from Level 4 uploaded to the Jobs site.
 *[draft published outputs]: a collection of released outputs prepared for publication to the general public.
-*[Data Builder]: a program that uses a data definition to create a datasets for analysis
-*[dataset definition]: specifies the details of data — for example, patients — that you want to include in your study. Used by Data Builder.
-*[ehrQL]: electronic health record query language. The language used by the Data Builder tool to extract cohorts
+*[Data Builder]: A program that uses data definitions to create datasets for analysis.
```

Comment:
minor point: Should `data definitions` be `dataset definitions` for consistency below?

larger point: I wonder if the phrase "data/dataset definitions" needs to be here at all. There's a danger, I think, of having too many interrelated definitions and that getting confusing. Could we get away with something like?
```
*[Data Builder]: A program which fetches the data needed for an analysis
```

Reply:
Good spot! It should be "dataset definitions" for consistency. Your larger point is also worth reflecting on: possibly as part of [sc-712](https://app.shortcut.com/ebm-datalab/story/712/update-glossary).